xml/monitoring.xml

```diff
@@ -29,7 +29,7 @@
   </dm:docmanager>
  </info>
   <sect1 xml:id="sec-remote-conman">
-  <title>ConMan &mdash; The Console Manager</title>
+  <title>ConMan &mdash; the console manager</title>
```

Comment:
```suggestion
  <title>ConMan &mdash; the CONsole MANager</title>
```

Reply:
Only a suggestion. It's consistent but it's not exactly pretty.

Reply:
Yeah, in this case, I'd leave it be. For MUMPS, we did this, because the acronym is both horrible and strangely non-obvious. Here, at least the acronym is obvious.